docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-REGEXFINDSET.xml

```diff
@@ -67,6 +67,8 @@
   role="underline">http://userguide.icu-project.org/strings/unicodeset</emphasis>.
   We use version 2.6 which should support all listed features.</para>
 
+  <para>REGEXFINDSET ignores capture groups. </para>
```

Comment:
I think it would be worthwhile to expand on this just a bit by adding, "REGEXFINDSET repeatedly extracts the text matching the entire regex pattern."  Or something to that affect.